code-scanning/eslint.yml

```
@@ -36,6 +36,8 @@ jobs:
           npm install @microsoft/eslint-formatter-sarif@2.1.7
```

Earlier versions of the formatter do not ignore suppressed alerts.

```suggestion
          npm install @microsoft/eslint-formatter-sarif@3.1.0
```